inc/cleantalk-pluggable.php

```diff
@@ -1439,11 +1439,19 @@ class_exists('Cleantalk\Antispam\Integrations\CleantalkInternalForms')
             return 'Plugin Name: WooCommerce Product Enquiry Premium - have the direct integration';
         }
 
-        // WP Discuz skip service requests. The plugin have the direct integration
+        // CoBlocks requests. The plugin have the direct integration
```

Comment:
Change this back to WP Discuz

Reply:
Thanks. Done https://github.com/CleanTalk/wordpress-antispam/pull/485/commits/67441afeac027de2211b5705fdc5e3424b08be5f